.eslintrc.json

```diff
@@ -5,31 +5,42 @@
         "browser": true,
         "mocha": true
     },
-    "extends": [
-        "xo/esnext",
-        "xo-react"
-    ],
+    "extends": ["airbnb"],
+    "plugins": ["jest"],
     "rules": {
-        "arrow-parens": ["error", "as-needed", {
-            "requireForBlockBody": true
-        }],
+        "arrow-parens": ["error", "as-needed", { "requireForBlockBody": true }],
+        "comma-dangle": ["error", "never"],
+        "import/extensions": ["error", "never", { "json": "always" }],
+        "import/no-extraneous-dependencies": "off",
+        "import/no-unresolved": ["error", { "ignore": ["frontend"] }],
         "indent": ["error", 4, {
             "SwitchCase": 1
         }],
-        "key-spacing": ["error", {
```

Comment:
why no key spacing enforcement?

Reply:
what do you mean? This enforces aligning object key-values on their colon. i.e.

```js
{
    hello       : 1
    longArgument: 2
}
```

That might prove to be really annoying, in which case we can undo it